examples/multiplayer-editor/jamsocket.config.json

```diff
@@ -1,4 +1,7 @@
 {
-  "dockerfile": "./src/session-backend/Dockerfile.jamsocket",
-  "watch": ["./src/session-backend"]
+  "dockerfile": "./src/session-backend/Dockerfile",
+  "watch": ["./src/session-backend"],
+  "dockerOptions": {
+    "path": "."
```

Comment:
why do we need the docker options?

Reply:
The `dockerOptions.path` value is what sets the context for the docker build command. It's the same as the `context` flag. (Some more information about that in [this section](https://docs.docker.com/reference/cli/docker/image/build/#build-with-path))